gitlens/gl-gitkraken-browser-extension.md

```diff
@@ -0,0 +1,58 @@
+---
+
+title: GitKraken Browser Extension
```

Comment:
Did we want to gear this doc more toward GitLens specifically, or are we ok with it including both clients? Cool with it either way, just want to confirm since this is going into GitLens docs.

Reply:
that is a great question that I am not sure of 😅 because of the current functionality (and time constraints), I opted to simply use the same content for both [GKC](https://github.com/gitkraken/gitkraken-client-docs/pull/163) and GL, minus the gif showing it open in each client. However, i think it might make sense to make these pages client specific going forward.